lang/ilias_cs.lang

```diff
@@ -1960,7 +1960,6 @@ auth#:#login_page_switch_rte#:#Použít Rich-Text-Editor
 auth#:#lti_consumer_inactive#:#Uživatel nástroje LTI je vypnutý.
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#Nyní online
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.